ui/js/controllers/perf/compare.js

```diff
@@ -197,34 +216,68 @@ perf.controller('CompareResultsCtrl', [
 
                     cmap.links = [];
 
-                    if (testName.indexOf("summary") > 0) {
-                        var detailsLink = 'perf.html#/comparesubtest?';
-                        detailsLink += $httpParamSerializer({
-                            originalProject: $scope.originalProject.name,
-                            originalRevision: $scope.originalRevision,
-                            newProject: $scope.newProject.name,
-                            newRevision: $scope.newRevision,
-                            originalSignature: oldSig,
-                            newSignature: newSig,
-                            framework: $scope.filterOptions.framework.id
-                        });
+                    if ($scope.originalRevision !== undefined){
```

Comment:
Just `if ($scope.originalRevision) {` is fine here (also note the space between `)` and `{`)